dt-posts/dt-posts.php

```diff
@@ -1594,12 +1594,26 @@ public static function get_default_list_column_order( $post_type ){
         return $columns;
     }
 
+    private static function adjust_post_tile_labels( $tiles ): array {
```

Comment:
@kodinkat, instead of "adjust" can we call this function "translation_post_tile_labels"?